src/screens/FeedScreen/index.js

```diff
@@ -39,25 +40,27 @@ const FeedScreen = (props) => {
     bottom,
     loading,
     myProfile,
-    nextTargetFeed,
-    postOffset,
     searchHeight,
     showNavbar,
+    timer,
+    viewPostTimeIndex,
 
     checkCacheFeed,
+    fetchNextFeeds,
     getDataFeeds,
     handleScroll,
-    isSamePostViewed,
     onBlockCompleted,
     onDeleteBlockedPostCompleted,
     saveSearchHeight,
     sendViewPostTime,
     setDownVote,
     setIsLastPage,
     setShowNavbar,
-    setUpVote,
-    updateViewPostTime
+    setUpVote
   } = useCoreFeed();
+
+  const {onWillSendViewPostTime} = useViewPostTimeHook(dispatch, timer, viewPostTimeIndex);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes here seem to be a refactoring of the destructuring assignment for the return value of `useCoreFeed()`. It appears that some variables have been removed (`nextTargetFeed`, `postOffset`, `isSamePostViewed`, `updateViewPostTime`) and others have been added (`timer`, `viewPostTimeIndex`, `fetchNextFeeds`, `setDownVote`, `setIsLastPage`, `setShowNavbar`, `setUpVote`). Make sure that all removed variables are no longer needed in the code below and that all added variables are being used.



<!-- This is an auto-generated comment by OSS CodeRabbit -->